### PR TITLE
ln: adds the syntax to create multiple symbolic links

### PR DESCRIPTION
**Pull request by jk**
----

~~- [ ] The page (if new), does not already exist in the repo.~~

~~- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      "command: add page" for new pages, or "command: description of changes" for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/ln.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 `ln -s {{path/to/file}} {{path/to/symlink}}`
 
+- Create several symbolic links to one file (or folder):
+
+`ln -s {{path/to/file}} {{path/to/symlink1}}`
```
Comment (agnivade) [inline pages/common/ln.md]: Sorry, maybe I am misunderstanding something. Isn't this exactly the same as the previous example ? You seem to have just changed `symlink` to `symlink1`.

Comment (jk) [reply pages/common/ln.md]: @agnivade Sorry, you're right, my bad. I just forgot the additional symlinks. I pushed the fix to my branch. Should be fine now.

**Comment by agnivade** [comment]
@jk - Thanks for the correction. Though I am not very sure if this is worth adding. We can easily run the first command multiple times to generate multiple symlinks. I will defer to others for the final decision.

/cc @sbrl , @waldyrious

**Comment by jk** [comment]
@agnivade Alright, no problem, thanks for the feedback.

**Comment by waldyrious** [comment]
@jk thanks for the contribution. I'm also hesitant about this -- could you please explain the motivation / use case for this invocation? We tend not to include too obscure usages of a command that most users won't be exposed to, to keep the signal-to-noise ratio up.

**Comment by jk** [comment]
@waldyrious My use case is that I create several symbolic links to a newly deployed directory during a deployment. But @agnivade's objection is justified, it is equally possible to simply apply the command several times in succession. I'm closing the pull request.

**Comment by waldyrious** [comment]
I appreciate your understanding :+1: Cheers!

**Pull request closed without merging**